spec/swagger/swagger_doc_v2.yaml

```diff
@@ -465,6 +465,26 @@
           schema:
             type: boolean
             example: true
+        - name: filter[profile_id]
```

Comment:
Yes, the code which does the filtering is applied to both endpoints the `GET` and the `POST` one. I didn't realise the frontend was only using the `POST` version though!

Reply:
Yeah, we actually removed the use of the `GET /moves` in favour of that because of a legacy problem with where the locations are retrieved and stored 😞